docs/100-upgrade/migration-guides/3.7-3.8.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: 3.7 -> 3.8
+description:
+  This page lists the highlights for upgrading a project from Front-Commerce 3.7
+  to 3.8
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+<p>{frontMatter.description}</p>
+
+## Update dependencies
+
+Update all your `@front-commerce/*` dependencies to this version:
+
+```shell
+pnpm update "@front-commerce/*@3.8.0"
+```
+
+## Automated Migration
+
+We provide a codemod to automatically update your codebase to the latest version
+of Front-Commerce. This tool will update your code when possible and flag the
+places where you need to manually update your code (with `// TODO Codemod`
+comments).
+
+```shell
+pnpm run front-commerce migrate --transform 3.8.0
+```
+
+## Additional SEO meta data
+
```

Comment:
**issue:** for the reader it's not clear what this section is about

**suggestion:** add an intro paragraph (also, this might fuse with what @Kagaminara added in https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/3659
```suggestion
## Additional SEO meta data

In this version, we enriched GraphQL types to expose additional data used for SEO or optional UI purpose.
```